### DISCUSSION TITLE
Whisper broken after pip install whisper --upgrade

### DISCUSSION THREAD

**Discussion by jltchiu** [openai/whisper#120] [Q&A] [answered]
Hi, at 9:40 AM EST 9/25/2022, I did the update and `Successfully installed whisper-1.1.10`

Now when I do python import whisper, I get

```
>>> import whisper
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "...\whisper.py", line 69, in <module>
    libc = ctypes.CDLL(libc_name)
  File "...\Python37\lib\ctypes\__init__.py", line 364, in __init__
    self._handle = _dlopen(self._name, mode)
TypeError: LoadLibrary() argument 1 must be str, not None
```

Is there any dependency that also need to be installed together with this update?

**Comment by jongwook** [comment, marked as answer]
It's an unrelated package hosted at [PyPI](https://pypi.org/project/whisper/). Please run:

```bash
pip uninstall whisper
pip install git+https://github.com/openai/whisper.git 
```

**Reply by wingsum93** [reply]
thank you so much. I solve my problem after following the instruction.

**Reply by MANAS8991** [reply]
but after follow this also not resolve the problem

**Comment by Nixxs** [comment]
this is a good solution but I was hoping to package openai-whisper into a pypi package im building at the moment and its currently not possible to pip install it yet. when do we think the current git version will be published?

**Reply by hashirscam** [reply]
are you done with it?
